### ISSUE TITLE
Reg\Events - External User can view closures & temporary shutdowns

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1709]
#### Description:
As an External User, I want the ability to view the grid of closures and temporary shutdowns for my operator

#### Acceptance Criteria:

Given that I am on the main dashboard OR registration dashboard
When I click Closures, Temporary Shutdowns, Restart Operations
Then I am brought to the Closures, Temporary Shutdowns, Restart Operations grid
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=15351-683&t=lsl3ZaYZigOiKk3a-4)

If the event was reported for an operation, then the facility column shows "N/A"
If the event was reported for a facility, then the operation column shows "N/A"

#### Development Checklist:

- [ ] query event table, but with filter so that the external user can only see the events relevant to their operator FOR THE TIME/S that the operation/facility was being controlled by their operator (i.e., query either `operation_operator_timeline` or `facility_operation_timeline`). There should be another filter to only retrieve events that are closures, temporary shutdowns, or restarts (i.e., no transfers)
- [ ] if an event is created for an LFO, the grid should display that event type for each facility within the LFO
- [ ] Paginate the endpoint (see `@paginate` decorator)
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by 
- Blocking #2000

**Comment by Sepehr-Sobhani** [comment]
Do we need to add `Partially Restarted` status as part of this ticket too?
